app/helpers/notification_subscriptions_helper.rb

```diff
@@ -1,7 +1,8 @@
 module NotificationSubscriptionsHelper
-  def render_notification_subscription_component(subscription_types)
+  def render_notification_subscription_component(subscription_types, journey_id = nil)
```

Comment:
Dajme tam nie len id ale journey objekt. (Primitive obsession), taktiez asi ako keyword argument ked to je optional. `journey: nil`